src/components/SideBar/index.tsx

```diff
@@ -25,7 +25,7 @@ class SideBar extends React.Component<SideBarProps> {
     return (
       <div className="background">
```

Comment:
I think u can remove this if u changed the name to `container` which seems to be like some sort of html keyword